src/utils/theme.js

```diff
@@ -57,16 +69,15 @@ export default {
     long: '450ms'
   },
   colors: {
-    white: '#fff',
-    text: '#333333',
-    textLight: '#828282',
-    secondaryText: '#757575',
-    vibrant: '#65FFCD',
-    dark: '#232323',
-    black: '#1d1d1d',
-    lightGrey: '#737272',
-    greyBG: '#f9f9f9',
-    grey: '#e6e6e6',
-    blueBg: '#090329'
+    blueBg: colorsVariables.darkBlue,
+    vibrant: colorsVariables.neonGreen,
+    white: colorsVariables.white,
+    greyBG: colorsVariables.lightGrey,
```

Comment:
secondarytext and light grey were 99% the same color (checked on a color comparison tool, `#757575` vs `#737272`) so I kept just one of them